include/gamepad/controller.hpp

```diff
@@ -16,31 +17,127 @@ enum EventType {
     ON_PRESS,
     ON_LONG_PRESS,
     ON_RELEASE,
+    ON_SHORT_RELEASE,
 };
 
 class Button {
         friend class Controller;
     public:
+        /**
+         * @brief Whether the button has just been pressed
+         *
+         */
```

Comment:
you don't need to use @brief for member vars

Reply:
Fixed in https://github.com/LemLib/Gamepad/pull/6/commits/402a0902d61fc883336be658d83bb0af32065a6b.